OutOfSchool/OutOfSchool.DataAccess/Repository/NotificationRepository.cs

```diff
@@ -25,17 +25,11 @@ public NotificationRepository(OutOfSchoolDbContext dbContext)
     /// <inheritdoc/>
     public async Task ClearNotifications()
     {
-        // TODO: replace it in EFCore7:
-        // await db.Notifications.Where(x => x.ReadDateTime == null && x.CreatedDateTime < dateNotReaded).ExecuteDeleteAsync();
-        // await db.Notifications.Where(x => x.ReadDateTime != null && x.CreatedDateTime < dateReaded).ExecuteDeleteAsync();
-        var dateNotReaded = DateTimeOffset.UtcNow.AddYears(-1);
-        var dateReaded = DateTimeOffset.UtcNow.AddMonths(-1);
-
-        await db.Database.ExecuteSqlRawAsync(
-            @"DELETE FROM Notifications WHERE ReadDateTime IS NULL AND CreatedDateTime < {0};
-            DELETE FROM Notifications WHERE ReadDateTime IS NOT NULL AND CreatedDateTime < {1};",
-            dateNotReaded,
-            dateReaded);
+        var dateWasNotRead = DateTimeOffset.UtcNow.AddYears(-1);
+        var dateWasRead = DateTimeOffset.UtcNow.AddMonths(-1);
+
+        await db.Notifications.Where(x => x.ReadDateTime == null && x.CreatedDateTime < dateWasNotRead).ExecuteDeleteAsync();
+        await db.Notifications.Where(x => x.ReadDateTime != null && x.CreatedDateTime < dateWasRead).ExecuteDeleteAsync();
```

Comment:
(Maybe performance) I wonder why this couldn't be done in one query? I mean usage of 
```
    .Where(x => (x.ReadDateTime == null && x.CreatedDateTime < dateWasNotRead) 
        || (x.ReadDateTime != null && x.CreatedDateTime < dateWasRead)) 
```

Reply:
It was only for readability.

Reply:
Now we have (sligtly better) readability with worse performance. Is it worth it?